ivre/db/mongo.py

```diff
@@ -2028,6 +2041,12 @@ def parse_args(self, args, flt=None):
             flt = self.flt_and(flt, self.searchsource(args.source))
         if args.timeago is not None:
             flt = self.flt_and(flt, self.searchtimeago(args.timeago))
+        if args.id is not None:
```

Comment:
You will have to update that also: because of the `nargs="+"`, `args.id` will be a list when using `argparse`, and a single value when using `optparse`.